pkg/util/traceid.go

```diff
@@ -9,7 +9,7 @@ import (
 	"unsafe"
 )
 
-func HexStringToTraceID(id string) ([]byte, error) {
+func HexStringToID(id string, isSpan bool) ([]byte, error) {
```

Comment:
The function `HexStringToID()` is no longer used outside this package (and probably should no longer be used, as it has merely become a helper function). Consider making the function private and maybe move it below the functions `HexStringToTraceID()` and `HexStringToSpanID()`